mobile-center-crashes/android/src/main/java/com/microsoft/azure/mobile/react/crashes/RNCrashesModule.java

```diff
@@ -114,12 +111,58 @@ public void run() {
     }
 
     @ReactMethod
-    public void crashUserResponse(boolean send, ReadableMap attachments, Promise promise) {
-        int response = send ? Crashes.SEND : Crashes.DONT_SEND;
-        if (mCrashListener != null) {
-            mCrashListener.setAttachments(attachments);
+    public void notifyWithUserConfirmation(int userConfirmation) {
+        Crashes.notifyUserConfirmation(userConfirmation - 1);
```

Comment:
Wouldn't it be safer to use a switch statement? In the future, if we change the numbers for whatever reason, it would be such a pain to find every time we use them in wrapper sdks and update them

Reply:
This `userConfirmation - 1` is really confusing without understanding Android SDK implementation. In Android SDK, we use 0 for `SEND`, 1 for `DONT_SEND` and 2 for `ALWAYS_SEND`, but in RN SDK we use 1 for `Send`, 2 for `DontSend` and 3 for `AlwaysSend`.

What's the reason we can't use `{Send: 0, DontSend: 1, AlwaysSend: 2}` for RN SDK? Can we keep them consistent?

Reply:
iOS is inconsitent with Android already.